modules/apps/frontend-js/frontend-js-loader-support/src/main/java/com/liferay/frontend/js/loader/support/internal/JSLoaderSupportImpl.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.loader.support.internal;
+
+import com.liferay.frontend.js.loader.support.JSLoaderSupport;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.servlet.taglib.aui.ScriptData;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Iván Zaera Avellón
+ */
+@Component(service = JSLoaderSupport.class)
+public class JSLoaderSupportImpl implements JSLoaderSupport {
+
+	@Override
+	public String getScriptBody(
+		String javascriptModule, String javascriptVariable,
+		String javascriptCode) {
+
+		StringBundler javascriptSB = new StringBundler(10);
+
+		javascriptSB.append("(function() {");
+
```

Comment:
What's with this empty line?

Reply:
Is starts a block closed in line 51, but it will be merged altogether anyway (in the final revision :tm: ), until we cannot understand what we are really writing, so I will do it myself before forwarding.

![elmo fire](https://media.giphy.com/media/e5kbmb3wX3J1S/giphy.gif)

Reply:
😂